### PR TITLE
simplify add/remove markup, set types to button to prevent enter caus…

### PR DESCRIPTION
**Pull request by charlienparker**
…ing form show

# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [x] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [x] Any endpoint modified in the PR only responds to the expected MIME types.
- [x] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [x] There are no inline styles added
- [x] There are no inline javascript added
- [x] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [x] Code does not use .html_safe
- [x] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: 
1. https://www.pivotaltracker.com/story/show/188305603
2. https://www.pivotaltracker.com/story/show/188305604

# A brief description of the changes

Current behavior: Enter key presses in the Manage Family and Consumer Registration forms cause the mailing form to appear.

New behavior: Enter key presses in the Manage Family and Consumer Registration forms do not cause the mailing form to appear.

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name: `BS4_CONSUMER_FLOW_IS_ENABLED`

- [ ] DC
- [x] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

### CODE REVIEW

File: app/javascript/css/colors.scss [hunk 1]
```diff
@@ -100,9 +100,9 @@
   --input-text-color: #454545;
   --form-element-text-color: #454545;
 
-  --button-default-color: var(--primary-color);
-  --button-default-hover-color: var(--primary-hover-color);
-  --button-default-active-color: var(--primary-color-darker);
+  --button-default-color: #a21e1f;
```
Comment (kristinmerbach) [inline app/javascript/css/colors.scss:103]: we are making all the primary buttons red?

Comment (charlienparker) [reply app/javascript/css/colors.scss:103]: negative

I had made this change correctly twice before, but had to completely wipe my local repo twice as I was dealing with some git shenanigans

**Review by TristanB17 — APPROVED** [review]
A few tests failing as of this review, but they seem to be `the usual suspects` that will pass once rerun: otherwise, lgtm!

**Review by kristinmerbach — APPROVED** [review]
LGTM

**Pull request merged**